pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PackageSettingsIntegrationTest.groovy

```diff
@@ -1,594 +0,0 @@
-/*
```

Comment:
See the comment below on the new tests on why I think they do not completely replace integration tests and by their nature cannot replace them fully.